common/persistence/src/test/java/app/coronawarn/server/common/persistence/domain/DiagnosisKeyTest.java

```diff
@@ -87,4 +88,30 @@ void testRetentionThresholdRejectsNegativeValue(int daysToRetain) {
     assertThatIllegalArgumentException()
         .isThrownBy(() -> diagnosisKey.isYoungerThanRetentionThreshold(daysToRetain));
   }
+
+
+  //Extra tests by Hallon, follows grammar above.
+
+  @Test
+  void testNullDoesNotEqual() {
+    assertThat(diagnosisKey.equals(null).isFalse);
```

Comment:
This syntax is incorrect. You could probably do something like `assertThat(diagnosisKey.equals(null)).isFalse()`, but something like (not sure if this exact syntax exitsts) `assertThat(diagnosisKey).isNotNull()` would be clearer.